tests/test_encode.py

```diff
@@ -51,6 +51,15 @@ def test_count_encode(t_train, t_test):
     assert res.to_pandas().sort_values(by="time").ticker.to_list() == [4, 4, 2, 2, 0, 0]
 
 
+def test_ordinal_encode(t_train, t_test):
+    step = ml.OrdinalEncode("ticker")
+    step.fit_table(t_train, ml.core.Metadata())
+    res = step.transform_table(t_test).order_by("time")
+    ordered_map = (("AAPL", 0), ("GOOG", 1), ("MSFT", 2))
```

Comment:
```suggestion
    ticker_encoding_map = (("AAPL", 0), ("GOOG", 1), ("MSFT", 2))
```
Maybe? `ordered_map` doesn't mean much here (actually, `ordered_map` seems more like an ordered map data structure).